### ISSUE TITLE
SC 135 -- Autocomplete is required on common form elements

### ISSUE THREAD

**Issue by damiansian** [newjersey/accessibility-pm#20]
## URL
https://demo.fileyourstatetaxes.org/en/questions/phone-number

## Issue
To come. 

## Screen shot
![Image](https://github.com/user-attachments/assets/8ac30086-df7b-4459-b664-f91e7c01603f)

## Video walkthrough
https://github.com/user-attachments/assets/fe747fa2-015d-49cf-a5a8-8acf567116ac

## WCAG Success Criterion
[SC 1.3.5: Identify Input Purpose (Level AA)](https://w3.org/TR/WCAG21#identify-input-purpose)

## Impacted users
To come. 

## Suggested fix
This suggestion fixes the issues on Mac and PC screen readers and brings the form element up to full WCAG conformance. There are other issues solved for here that have separate bugs filed. 

What changed: 
- Added autocomplete (SC 1.3.5)
- Associated helper text (SC 3.3.2)
- Added `required` since skipping entry causes and error (SC 3.3.1 and SC 3.3.2)
- Used `aria-describedby` for helper text AND error text. 

### Current
```html
<label for="state_file_phone_number_form_phone_number"><p class="form-question">Your phone number</p>
<p class="text--help">Enter numbers only</p>
</label>
<input 
    autocomplete="off" 
    autocorrect="off" 
    autocapitalize="off" 
    spellcheck="false" 
    type="text" 
    class="form-width--long text-input" 
    id="state_file_phone_number_form_phone_number" 
    name="state_file_phone_number_form[phone_number]">

    </div>
``` 

  
### Should be:
  ```html
<label id="label" for="phone">Your phone number</label>
<input 
    type="text" 
    name="phone" 
    id="phone" 
    autocomplete="tel" 
    aria-describedby="helper"
    aria-required="true">
<p id="helper">Enter numbers only</p>
</div>
```

**Comment by damiansian** [comment]
Looks good

<img width="700" alt="Autofill picks up phone number. " src="https://github.com/user-attachments/assets/c169082a-5aa7-466c-af37-f76fda3ef565" />
